akka-http/server/src/main/scala/endpoints/akkahttp/server/Urls.scala

```diff
@@ -33,6 +35,8 @@ trait Urls extends algebra.Urls {
   // Query strings
   //***************
 
+  implicit lazy val uuidQueryString: QueryStringParam[UUID] = Unmarshaller.strict[String, UUID](UUID.fromString)
```

Comment:
Yes, this could be rewritten like this:

```
  implicit lazy val uuidQueryString: QueryStringParam[UUID] = Unmarshaller(
    implicit ec =>
      (s: String) =>
        Try(Future(UUID.fromString(s))).fold(Future.failed, identity))
```